.github/workflows/conformance-runtime.yaml

```diff
@@ -0,0 +1,422 @@
+        run: |
+          cat > test/cilium-ssh-config.txt << EOF
+          Host runtime
+            HostName 127.0.0.1
+            User root
+            Port 2222
+            UserKnownHostsFile /dev/null
+            StrictHostKeyChecking no
+            PasswordAuthentication no
+            LogLevel FATAL
+          EOF
+          cd test
+          export INTEGRATION_TESTS=true
+          ./test.test \
+          --ginkgo.focus="${{ steps.vars.outputs.focus }}" \
+          --ginkgo.skip="${{ matrix.skip }}" \
+          --ginkgo.seed=1679952881 \
+          --ginkgo.v -- \
+          -cilium.provision=false \
+          --cilium.image=quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/cilium-ci \
+          -cilium.tag=${{ steps.vars.outputs.sha }}  \
+          --cilium.operator-image=quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/operator \
+          -cilium.operator-tag=${{ steps.vars.outputs.sha }} \
+          --cilium.hubble-relay-image=quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/hubble-relay-ci \
+          -cilium.hubble-relay-tag=${{ steps.vars.outputs.sha }} \
+          -cilium.operator-suffix=-ci \
+          -cilium.SSHConfig="cat ./cilium-ssh-config.txt"
+
+      - name: Runtime privileged tests
+        if: ${{ matrix.focus == 'f-03-privileged' }}
+        timeout-minutes: 20
+        uses: cilium/little-vm-helper@0b7d7157dae56a44dc531c852e7756dc671071af # v0.0.3
+        with:
+          provision: 'false'
+          cmd: |
+            cd /host
+            # Install go-junit-report to generate junit files for the
+            # privileged tests.
+            go install github.com/jstemmer/go-junit-report/v2@7fde4641acef5b92f397a8baf8309d1a45d608cc
+            export GOTEST_FORMATTER="/root/go/bin/go-junit-report -set-exit-code -iocopy -out test/privileged-runtime.xml"
+            make tests-privileged NO_COLOR=1
+
+      - name: Fetch artifacts
+        if: ${{ !success() && (matrix.focus == 'f-01-agent' || matrix.focus == 'f-02-datapath') }}
+        shell: bash
+        run: |
+          tar -zcf test_results-${{ matrix.name }}.tar.gz ./test/test_results
+
+      - name: Upload artifacts
+        if: ${{ !success() && (matrix.focus == 'f-01-agent' || matrix.focus == 'f-02-datapath') }}
+        uses: actions/upload-artifact@0b7f8abb1508181956e8e162db84b466c27e18ce # v3.1.2
+        with:
+          name: cilium-sysdumps
+          path: |
+            test_results-*.tar.gz
+          retention-days: 5
+
+      - name: Fetch JUnits
+        if: ${{ always() }}
+        shell: bash
+        run: |
+          mkdir -p cilium-junits
+          cd test/
+          for filename in *.xml; do cp "${filename}" "../cilium-junits/${{ matrix.focus }}-${filename}"; done;
+
+      - name: Upload JUnits
+        if: ${{ always() }}
+        uses: actions/upload-artifact@0b7f8abb1508181956e8e162db84b466c27e18ce # v3.1.2
+        with:
+          name: cilium-junits
+          path: |
+            cilium-junits/*.xml
+          retention-days: 2
```

Comment:
That's not enough to get statistics for http://isogo.to/dashboard-ci AFAIK. What else is needed? Not having those statistics will make the CI Health job a lot harder (as it already is for existing GitHub workflows :disappointed:).

Reply:
Nice! IIRC the Slack message, this is only for `main` (vs. PR runs) right now, right? It's not ideal, but I think it's enough to alleviate my concerns regarding CI Health debugging.

Reply:
Correct, only for `main` and stable branches.

Reply:
FYI, looking at the last table from the screenshot, the JUnit looks pretty "broken" (i.e., listing each small step through the test?).